ClangSharpPInvokeGenerator/Cursors/TranslationUnit.cs

```diff
@@ -19,11 +19,6 @@ protected override CXChildVisitResult VisitChildren(CXCursor childHandle, CXCurs
         {
             ValidateVisit(ref handle);
 
-            if (!childHandle.Location.IsFromMainFile)
```

Comment:
We need the traversal to actually cover all nodes, including those not in the main file. This is important for handling things like base class declarations that exist in an included file.